controller/sync_strats/rtx.go

```diff
@@ -66,22 +64,26 @@ func (rtx *RouterSender) GetState() env.RouterStateValues {
 
 func (rtx *RouterSender) Start() {
 	if rtx.running.CompareAndSwap(false, true) {
```

Comment:
I'd leave closeNotify init above. I don't see a reason to create it just-in-time or nil it out afterwards. Maybe I'm missing something, though?

Reply:
It was to allow the `RouterSender` to be re-stoppable and re-startable. I can make it not that as that functionality isn't used in the final version.

Reply:
I'd wait until we're using it to add that, just in case we never do :)

Reply:
Removed.